tests/integration/sdk_api/alerts/test_alerts.py

```diff
@@ -228,6 +228,92 @@ def test_alert_operations(self):
             alerts_activations = self._get_alert_activations(project_name)
             assert len(alerts_activations) == 0
 
+    def test_alert_activations_sdk(self):
+        project_name = "my-new-project"
+        project = mlrun.new_project(project_name)
+
+        activations = project.list_alert_activations()
+        assert len(activations) == 0
+
+        activations, token = project.paginated_list_alert_activations()
+        assert len(activations) == 0
+        assert token is None
+
+        event_name_entity_list = [
+            (
+                "alert1",
+                alert_objects.EventKind.DATA_DRIFT_DETECTED,
+                alert_objects.EventEntityKind.MODEL_ENDPOINT_RESULT,
+            ),
+            (
+                "alert2",
+                alert_objects.EventKind.FAILED,
+                alert_objects.EventEntityKind.JOB,
+            ),
+        ]
+
+        alert_summary = "Alert summary"
+
+        for alert_name, event_name, alert_entity in event_name_entity_list:
+            self._create_alert(
+                project_name,
+                alert_name,
+                alert_entity,
+                project_name,
+                alert_summary,
+                event_name,
+                reset_policy=mlrun.common.schemas.alert.ResetPolicy.AUTO,
+            )
+
+        iterations = 5
+        for i in range(iterations):
+            for alert_name, event_name, alert_entity in event_name_entity_list:
+                self._post_event(project_name, event_name, alert_entity)
+
+        # check regular SDK without filters
+        activations = project.list_alert_activations()
+        assert len(activations) == 2 * iterations
+
+        activations, token = project.paginated_list_alert_activations(
+            page_size=iterations
+        )
+        assert len(activations) == iterations
+
+        activations, token = project.paginated_list_alert_activations(page_token=token)
+        assert len(activations) == iterations
```

Comment:
This is the last page, so the token should be None here, right?

Reply:
@moranbental right, sorry, that's leftovers from the tests I showed you recently, will remove this check